xrm/index.d.ts

```diff
@@ -205,16 +214,194 @@ declare namespace Xrm
              */
             openWebResource( webResourceName: string, webResourceData?: string, width?: number, height?: number ): Window;
         }
+
+        /**
+          * Provides methods to create and manage records in the mobile clients (for phones tablets).
+          */
+        Mobile: {
```

Comment:
Sure, I'll make that change. Any thoughts on the changes to the Control interface?